public/v4/apps/novu.yml

```diff
@@ -0,0 +1,227 @@
+        image: $$cap_NOVU_IMAGE_URL/ws:$$cap_NOVU_IMAGE_VERSION
+        environment:
+            PORT: $$cap_NOVU_PORT_WEBSOCKET
+            NODE_ENV: $$cap_ENVIRONMENT
+            MONGO_URL: mongodb://srv-captain--$$cap_appname-db/$$cap_MONGO_DB
+            REDIS_HOST: srv-captain--$$cap_appname-cache
+            REDIS_PORT: $$cap_REDIS_PORT
+            JWT_SECRET: $$cap_NOVU_JWT_SECRET
+        depends_on:
+            - $$cap_appname-db
+            - $$cap_appname-cache
+
+    $$cap_appname:
+        caproverExtra:
+            containerHttpPort: $$cap_NOVU_PORT_APPLICATION
+        image: $$cap_NOVU_IMAGE_URL/web:$$cap_NOVU_IMAGE_VERSION
+        environment:
+            REACT_APP_API_URL: http://$$cap_appname-api.$$cap_root_domain
+            REACT_APP_ENVIRONMENT: $$cap_ENVIRONMENT
+            REACT_APP_WIDGET_EMBED_PATH: http://$$cap_appname-embed.$$cap_root_domain/embed.umd.min.js
+            REACT_APP_DOCKER_HOSTED_ENV: $$cap_NOVU_DOCKER_HOSTED_ENVIRONMENT
+        depends_on:
+            - $$cap_appname-api
+
+    $$cap_appname-widget:
+        caproverExtra:
+            containerHttpPort: $$cap_NOVU_PORT_WIDGET
+        image: $$cap_NOVU_IMAGE_URL/widget:$$cap_NOVU_IMAGE_VERSION
+        environment:
+            REACT_APP_API_URL: http://$$cap_appname-api.$$cap_root_domain
+            REACT_APP_WS_URL: http://$$cap_appname-ws.$$cap_root_domain
+            REACT_APP_ENVIRONMENT: $$cap_ENVIRONMENT
+        depends_on:
+            - $$cap_appname-api
+            - $$cap_appname
+
+    $$cap_appname-embed:
+        caproverExtra:
+            containerHttpPort: $$cap_NOVU_PORT_EMBED
+        image: $$cap_NOVU_IMAGE_URL/embed:$$cap_NOVU_IMAGE_VERSION
+        environment:
+            WIDGET_URL: http://$$cap_appname-widget.$$cap_root_domain
+        depends_on:
+            - $$cap_appname-widget
+
+caproverOneClickApp:
+    displayName: Novu
+    isOfficial: true
+    description: Open Source Notification Infrastructure For Developers
+    documentation: https://docs.novu.co
+    instructions:
+        start: |-
+            Novu is a fully functional real-time notification center for your web and react apps.
+            The only open-source notifications infrastructure that manages multi-channel content, scheduled notifications, digest engine, user preferences, and delivers Email, SMS, Push and Chat notifications using a single API.
+        end: |-
+            Novu has been successfully deployed! It might take few moments before it's fully started.
+            You can access it at `http://$$cap_appname.$$cap_root_domain`.
+    variables:
+        - id: $$cap_NOVU_IMAGE_VERSION
+          label: Application | Version
+          description: Version tag of Novu's Docker images. Check out their valid tags at https://github.com/novuhq/novu/pkgs/container/novu%2Fapi
+          defaultValue: '0.8.0'
+          validRegex: /.{1,}/
+        - id: $$cap_NOVU_IMAGE_URL
```

Comment:
nit:
This seems like an overkill for a variable. It's best to keep variable list short to the ones that are typically changed for the end users. The image URL is most likely going to be fixed and not changed.

Reply:
Allright, I'll keep that in mind.
Thanks.